DiscordChatExporter.Core.Rendering/Resources/HtmlCore.css

```diff
@@ -57,6 +57,70 @@ img {
     border-radius: 3px;
 }
 
+.spoiler.hidden {
+    cursor: pointer;
+}
+
+.spoiler.hidden .spoilerText {
+    opacity: 0;
+}
+
+.spoiler-image {
+    margin-top: 0.3em;
+    border-radius: 3px;
+    -webkit-box-shadow: 0.5px 0.5px 1px 1px rgba(0,0,0,.1);
```

Comment:
`box-shadow` is supported by all browsers (even partially by IE9) so I suggest we drop the vendor extension.